### ISSUE TITLE
Could '.sqlite' extension be usable?

### ISSUE THREAD

**Issue by jgranduel** [proofrock/ws4sqlite#4]
As there is no official SQLite extension, could at least `.sqlite` be also used? (see for instance on StackOverFlow [does-it-matter-what-extension-is-used-for-sqlite-database-files](https://stackoverflow.com/questions/808499/does-it-matter-what-extension-is-used-for-sqlite-database-files) or  [what-is-the-difference-between-sqlite-and-db-file](https://stackoverflow.com/questions/5847067/what-is-the-difference-between-sqlite-and-db-file). And I happen to use `.sqlite` ;-) 
Thx

**Comment by proofrock** [comment]
Neat! Let me think about the impacts, the actual enforcing of this shouldn't be in so many parts of code. So I think it's definitely doable. Thanks!

**Comment by proofrock** [comment]
Yep, it should be enough to change `cli.go`, line 77, to

```go
if !strings.HasSuffix(dbFiles[i], ".db") && !strings.HasSuffix(dbFiles[i], ".sqlite") {
```

And write the tests ;-) I will include this in the next release. Or do you think that it would be more general to create a `--allow-any-file-ext` switch?

**Comment by proofrock** [comment]
Thinking twice, probably removing the check altogether could be the best idea.

**Comment by jgranduel** [comment]
I think `--allow-any-file-ext` option would be overkill. I recall having read something about common SQLite database extension on SQLite site, but cannot find anymore. Google returns 3 extensions [q=sqlite+database+extension](https://www.google.com/search?q=sqlite+database+extension) loaded from [Wikipedia](https://en.wikipedia.org/wiki/SQLite). Maybe check these 3 extensions (.db, .sqlite, .sqlite3), it might be sufficient. 
Best regards

**Comment by proofrock** [comment]
Fair enough, but since sqlite doesn't mandate an extension, I feel I shouldn't too. Thanks for pointing this out, will fix in the next release.

**Comment by proofrock** [comment]
Commit a988e9ce1d94f892faed08e3dfffb8a9d302d0d2

**Comment by proofrock** [comment]
Released in v0.11.3
